fbrp/examples/10_ifttt/ifttt.py

```diff
@@ -0,0 +1,19 @@
+import a0
+import signal
+import requests
+
+EVENT_NAME = a0.cfg(a0.env.topic(), "/ifttt/event_name", str)
+KEY = a0.cfg(a0.env.topic(), "/ifttt/key", str)
+EVENT_TOPIC = a0.cfg(a0.env.topic(), "/event_topic", str)
+a0.update_configs()
+
+
+def call_robo_crit_event(pkt):
+    print(f"Pinging IFTTT Event {EVENT_NAME}")
+    requests.post(f"https://maker.ifttt.com/trigger/{EVENT_NAME}/with/key/{KEY}")
```

Comment:
Not necessarily in this PR, but do you want to post the message content?

Reply:
It's pretty straightforward to include the message content as well, I can just extend the current one and add comment on how to use if the user does not want the message.